i6_models/parts/factored_hybrid/diphone.py

```diff
@@ -0,0 +1,145 @@
+__all__ = [
+    "FactoredDiphoneBlockV1Config",
+    "FactoredDiphoneBlockV1",
+]
+
+from dataclasses import dataclass
+from typing import Callable, Tuple, Union
+
+import torch
+from torch import nn, Tensor
+import torch.nn.functional as F
+
+from i6_models.config import ModelConfiguration
+
+from .util import BoundaryClassV1, get_center_dim, get_mlp
+
+
+@dataclass
+class FactoredDiphoneBlockV1Config(ModelConfiguration):
+    """
+    Attributes:
+        context_mix_mlp_dim: inner dimension of the context mixing MLP layers
+        context_mix_mlp_num_layers: how many hidden layers on the MLPs there should be
+        dropout: dropout probabilty
+        left_context_embedding_dim: embedding dimension of the left context
+            values. Good choice is in the order of num_contexts.
+        num_contexts: the number of raw phonemes/acoustic contexts
+        num_hmm_states_per_phone: the number of HMM states per phoneme
+        num_inputs: input dimension of the output block, must match w/ output dimension
+            of main encoder (e.g. Conformer)
+        phoneme_state_class: the phoneme state augmentation to apply
```

Comment:
```suggestion
        boundary_class: the phoneme state augmentation to apply
```

also make a conscious decision on the order of parameters and then order the docstring in the same order as the parameters defined pls.